src/layouts/Contact.astro

```diff
@@ -1,112 +1,64 @@
---- 
-import BasicMeta from '../components/BasicMeta.astro'
-import Header from '../components/Header.astro'
-import Footer from '../components/Footer.astro'
-import Global from '../components/Global.astro'
-const {title} = Astro.props;
 ---
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <Global />
-    <BasicMeta title={title} />
-</head>
-
-<body>
-    <Header />
-    <main>
-        <div role="presentation" id="loading-container">
-            <div id="loading"></div>
-        </div>
-
-        <slot/>
-    </main>
-    <Footer />
-    <script data-goatcounter="https://vivic.goatcounter.com/count" async src="//gc.zgo.at/count.js"></script>
-</body>
-</html>
+import MainLayout from './Main.astro'
+const { content } = Astro.props;
+---
+<MainLayout title={content.title}>
```

Comment:
I'm reusing `Main.astro` as the base layout, and letting the others inherit from it. They all duplicated the file and added a bit more, so we can reduce duplication and centralize the style.